sktime/distances/_numba_utils.py

```diff
@@ -135,10 +137,10 @@ def to_numba_pairwise_timeseries(x: np.ndarray) -> np.ndarray:
     num_dims = _x.ndim
     if num_dims == 1:
         shape = _x.shape
-        _x = np.reshape(_x, (1, 1, shape[0]))
+        _x = np.reshape(_x, (shape[0], 1, 1))
```

Comment:
I think the old version of this is correct? This is converting it to (m,1,1), should it not be (1,1,m)?

Reply:
I've merged this function into _make_3d_series and better documented why it is reshaping in this way.